src/SubFolder.php

```diff
@@ -24,79 +28,69 @@ final class SubFolder implements MiddlewareInterface
     /**
      * @param UrlGeneratorInterface $uriGenerator The URI generator instance.
      * @param Aliases $aliases The aliases instance.
-     * @param string|null $prefix URI prefix the specified immediately after the domain part.
-     * The prefix value usually begins with a slash and must not end with a slash.
-     * @param string|null $alias The path alias {@see Aliases::get()}.
+     * @param string $baseUrlAlias The base url alias {@see Aliases::get()}. Default "@baseUrl".
      */
     public function __construct(
         private UrlGeneratorInterface $uriGenerator,
         private Aliases $aliases,
-        private ?string $prefix = null,
```

Comment:
Need keep ability to set prefix manually.

Reply:
Are there cases?

Reply:
1) Not totally 100% cases algorithm with autodetect base URL will be work (for example, inappropriate value in server params).

2) Performance.

Reply:
Well, what is the priority of these parameters?

Reply:
Value set manually has more priority.

Reply:
`/<prefix>/<@baseUrl>` ?

Reply:
If set prefix, then base URL equal to prefix.

```php
$baseUrl = $this->prefix ?? $this->getBaseUrl($request);
```